### PR TITLE
trello: fix double delete click

### PR DESCRIPTION
**Pull request by stevenao**

### CODE REVIEW

File: editor/d2l-rubric-criterion-editor.html [hunk 1]
```diff
@@ -403,15 +403,20 @@
 				return '';
 			},
 
-			_handleDeleteCriterion: function() {
+			_handleDeleteCriterion: function(e) {
 				var action = this.entity.getActionByName('delete');
 				if (action) {
+					var deleteButton = e.currentTarget;
+					var deleteButtonDisableState = deleteButton.disabled;
+					deleteButton.disabled = true;
```
Comment (emil-furniss) [inline editor/d2l-rubric-criterion-editor.html]: Why can't we just set `disabled = true` here, and then in the `finally` block `disabled = false`?

Comment (stevenao) [reply editor/d2l-rubric-criterion-editor.html]: yeah, was trying to play defensively. But now, I think if it was previously disabled. There won't be anything button click triggered anyway. Will update it.

Comment (emil-furniss) [reply editor/d2l-rubric-criterion-editor.html]: and we're sure that a disabled button doesn't fire events?

Comment (stevenao) [reply editor/d2l-rubric-criterion-editor.html]: yup, it won't trigger the event.

**Pull request merged**